src/gs-page.c

```diff
@@ -180,6 +184,9 @@ gs_page_app_installed_cb (GObject *source,
 		return;
 	}
 
+        now = g_date_time_new_now_local ();
+        g_settings_set (helper->settings, "install-timestamp", "x", g_date_time_to_unix (now));
+
```

Comment:
Why do you need to update the timestamp in the installed page? Is it just for consistency with the setting's name?

Reply:
Also notice that this line + their copies are not correctly indented (they're using spaces instead of tabs).

Reply:
Hi @joaquimrocha 
Maybe you didn't read my comment : https://github.com/endlessm/gnome-software/pull/386/files#r201407598 ?

Yes, this is not ideal but I am looking into if I can determine if that callback was in context of an update (and not in context of a installlation).

Reply:
Ah. Sorry, it wasn't clear to be. In any case, I'd rather we only add it here once we really find out if it's needed.